src/cpp/r/R/Tools.R

```diff
@@ -269,6 +269,17 @@ environment(.rs.Env[[".rs.addFunction"]]) <- .rs.Env
   devToolsPath %in% .libPaths()
 })
 
+.rs.addFunction("isDevPackage", function(name) {
+   # use pkgload if available
+   if (.rs.isPackageInstalled("pkgload")) {
```

Comment:
I think we should instead check `"pkgload" %in% loadedNamespaces()`. Presumedly, development packages could only be on the search path if `devtools::load_all()` was called, and so we could infer that if `pkgload` is not currently loaded, there won't be any dev packages.

Reply:
Or, in other words, if `pkgload` isn't already loaded, we shouldn't try to load it here -- we should just use it opportunistically if it's already loaded.